source/_docs/installation/updating.markdown

```diff
@@ -35,7 +35,7 @@ For a Raspberry Pi Docker container, simply pull the latest stable one:
 sudo docker pull homeassistant/raspberrypi3-homeassistant:stable
 ```
 
-After updating, you must start/restart Home Assistant for the changes to take effect. This means that you will have to restart `hass` itself.
+After updating, you must start/restart Home Assistant for the changes to take effect. This means that you will have to restart `ha` itself.
```

Comment:
This part of the documentation is about running Home Assistant Core directly in Python. `hass` is the correct command line here. This change is incorrect and needs to be reverted.